### PR TITLE
Use hook for press scale animation on buttons

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Adds a new generic hook `usePressScaleAnimation` to replace existing animation sequences for buttons

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested a settings button and my copy button from #213 on iOS simulator

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

N/A

### CODE REVIEW

File: src/hooks/usePressScaleAnimation.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import { Animated } from 'react-native'
+
+export const usePressScaleAnimation = (scaleTo = 0.97) => {
+  const scale = useRef(new Animated.Value(1)).current
+
+  const startPress = useCallback(() => {
+    Animated.timing(scale, {
+      toValue: scaleTo,
+      duration: 100,
```
Comment (raymondjacobson) [inline src/hooks/usePressScaleAnimation.ts:11]: Can we switch this to spring physics or use easing https://reactnative.dev/docs/easing#ease

**Review by sliptype — APPROVED** [review]
Cleeeean

File: src/hooks/usePressScaleAnimation.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/hooks/usePressScaleAnimation.ts:11]: and for web consistency, 
https://github.com/AudiusProject/stems/blob/master/src/components/Button/Button.module.css#L8
https://github.com/AudiusProject/stems/blob/master/src/assets/styles/animations.css#L9

File: src/components/audio-breakdown-drawer/AudioBreakdownDrawer.tsx [hunk 2]
```diff
@@ -316,7 +317,7 @@ const Wallet = ({ chain, address, balance }: WalletProps) => {
   const solWalletAudioEnabled = false
   const styles = useThemedStyles(createStyles)
 
-  const scale = useRef(new Animated.Value(1)).current
+  const { scale, startPress, stopPress } = usePressScaleAnimation(0.98)
```
Comment (raymondjacobson) [inline src/components/audio-breakdown-drawer/AudioBreakdownDrawer.tsx:320]: might be nice to have these `{ scale, onPressIn, onPressOut }` so it's super obvious where they should go

File: src/hooks/usePressScaleAnimation.ts (hunk 1 above)
Comment (rickyrombo) [reply src/hooks/usePressScaleAnimation.ts:11]: It's already using easing in-out by default, but if you want I can make it explicit?
https://reactnative.dev/docs/animated#configuring-animations
> Animated.timing() animates a value over time using easing functions.
> By default, it uses a symmetric easeInOut curve that conveys the gradual acceleration of an object to full speed and concludes by gradually decelerating to a stop.

Comment (raymondjacobson) [reply src/hooks/usePressScaleAnimation.ts:11]: nice nvm

**Review by raymondjacobson — APPROVED** [review]

**Pull request merged**